README.md

```diff
@@ -35,6 +34,17 @@ You can select to create a new note or continue working on a previous note.
 - Convert text to emoji using scroll bar in "Edit". Convert text such as ":smile:" to 😀, or ":eggplant:" to 🍆. Use shortcut "Ctrl-e" to convert text to emoji.
 - Continue where you last left off
 
+## Keyboard Shortcuts
+- `CTRL+K` Open Top Tool Bar
+- `ESC` Focus cursor on the text field
+- `CTRL+N` Start a new file
+- `CTRL+S` Save current file
+- `CTRL+O` Open an existing note
+- `CTRL+Q` Exit the application
+- `CTRL+A` Select Everything
+- `CTRL+Z` Undo
+- `CTRL+E` Turn text like `:smile:` into :smile:
+
```

Comment:
Theres also ctrl-x,c,v but idk if u want to document those because those are normal cut/copy/paste shortcuts

Reply:
Ctrl-c, x didn't work for me. 🤔

Reply:
Yeah I figured they are normal, and they also don't really fit the _vibe_ of a TUI to me, because highlighting, while possible, doesn't feel native to me.